contracts/ProviderPool.sol

```diff
@@ -4,8 +4,22 @@ import "./SortedDoublyLL.sol";
 import "zeppelin-solidity/contracts/ownership/Ownable.sol";
 
 contract ProviderPool is Ownable {
+
+  // TODO: init
+  uint public numberOfActiveProviders;
+
   using SortedDoublyLL for SortedDoublyLL.Data;
-  SortedDoublyLL.Data public providerPool;
+  SortedDoublyLL.Data internal providerPool;
+
+  function setProviderPoolMaxSize(uint _maxNumber) external onlyOwner {
+    // TODO: Remove limitation providerPool.maxSize = _maxNumber;
```

Comment:
I take it this is only for testing now, in practice there is no reason to limit the pool size right?

Reply:
For limiting gas costs. Since adding a new provider performs an inserting sort in the providerPool, the cost of adding a new provider linearly grows with the size of the pool.
It also affects the cost of `initializeRound()`

That is the main reason why Livepeer put a limit, so I did the same, but I can remove it if gas costs is not a concern